examples/pm2-typescript/tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
To follow Nuxt TS best practices, I suggest doing like https://github.com/nuxt/nuxt.js/blob/dev/examples/typescript/tsconfig.json
You can copy-paste and replace `@nuxt/typescript-edge` by `@nuxt/typescript` & `@nuxt/vue-app-edge` by `@nuxt/vue-app`.